db/data_migration/20180823093420_reslug_south_georgia_and_the_south_sandwich_islands.rb

```diff
@@ -0,0 +1,12 @@
+old_slug = "south-georgia-and-south-sandwich-islands"
+new_slug = "south-georgia-and-the-south-sandwich-islands"
+
+WorldLocation
+  .where(slug: old_slug)
+  .update_all(slug: new_slug)
+
+  Services.publishing_api.unpublish(
+    "ad4a059c-2cd7-4c49-9d0a-ba5f4b93bd72",
+    type: "redirect",
+    alternative_path: "/world/south-georgia-and-the-south-sandwich-islands/news",
+  )
```

Comment:
Moving this content should require a "put content" and "publish" in my mind, how come there is an "unpublish" request here?

Reply:
The unpublish redirects the old news page to the new one; this doesn't happen when you reslug the world location due to a bug. See https://github.com/alphagov/whitehall/pull/4137 for more details.

Reply:
I'm not sure it's a bug, but it's definitely something that can be improved, and I'd like to improve it before working around it here.

I've got one suggestion here, to store the content ids for the news pages in a similar way to the world locations themselves: https://github.com/alphagov/whitehall/pull/4293

Reply:
> I've got one suggestion here, to store the content ids for the news pages in a similar way to the world locations themselves: #4293

With help from Bevan, this is now merged. So hopefully once it's deployed, the unpublish command will be unnecessary.

Reply:
I've removed this